_posts/2019-12-16-Image-caching-with-FFImageloading.md

```diff
@@ -96,15 +96,16 @@ var image = new CachedImage
 
 | Property | Description |
 |-|-|
-|LoadingPlaceholder|If set, a loading placeholder is shown while loading. It supports UriImageSource, FileImageSource and StreamImageSource.|
- |ErrorPlaceholder|If set, an error placeholder is shown when error occurs while loading image. It supports UriImageSource, FileImageSource and StreamImageSource.|
- |RetryCount (int, default: 3)|If image download failed, or something wrong happened, it can be automatically retried. Defines retries count.|
- |RetryDelay (int, default: 250)|If image download failed, or something wrong happened, it can be automatically retried. Defines retry delay.|
+|`LoadingPlaceholder`|If set, a loading placeholder appears during loading. It supports `UriImageSource`, `FileImageSource`, and `StreamImageSource`.|
+ |`ErrorPlaceholder`|If set, an error placeholder appears when an error occurs while loading an image. It supports `UriImageSource`, `FileImageSource`, and `StreamImageSource`.|
+ |`RetryCount` (integer, default: 3)|Defines retries count. If an image download or other failure occurs, it can automatically be retried.|
+ |`RetryDelay` (integer, default: 250)|Defines retry delay. If an image download or other failure occurs, it can automatically be retried.|
  |CacheDuration (Timespan, default: `TimeSpan.FromDays(90)`)|Defines how long file cache of downloaded image is valid.|
 
 ## Clear cache on the memory and disk
  If there is not enough memory on a device, you need to suppress memory usage in your app, FFImageLoading provides a way to clear cache on the memory.
 
+(Question: Is the sentence "Provided in `CoreApplication`, it is the best place to clear memory cache" part of the text or code?)
```

Comment:
It is the comment text in the code. The sentence explains the code right below (`protected override void OnLowMemory(LowMemoryEventArgs e)`).